commit 6e0bef9fc97bd3af9d135d754341bb0349fc2bbf
Author: Pawel Moll <pawel.moll@st.com>
Date:   Fri May 23 16:01:57 2008 +0100

    mb671: Configure MII0 bus into RMII mode with MII_CLK.
    
    Signed-off-by: Pawel Moll <pawel.moll@st.com>

--- linux/arch/sh/boards/st/mb671/setup.c.orig	2008-05-26 10:21:31.000000000 +0200
+++ linux/arch/sh/boards/st/mb671/setup.c	2008-05-26 10:55:22.000000000 +0200
@@ -208,8 +208,12 @@ static int __init device_init(void)
 
 	stx7200_configure_sata(0);
 
-	stx7200_configure_ethernet(0, 0, 1, 0);
-//	stx7200_configure_ethernet(1, 0, 1, 1);
+#if 1 /* On-board PHY (MII0) in RMII mode, using MII_CLK */
+	stx7200_configure_ethernet(0, 1, 0, 0);
+#else /* External PHY board (MII1) in MII mode, using its own clock */
+	stx7200_configure_ethernet(1, 0, 1, 1);
+#endif
+
 	stx7200_configure_lirc(NULL);
 
 	return platform_add_devices(mb671_devices, ARRAY_SIZE(mb671_devices));
